pkg/controllers/agent/yurtcluster_controller.go

```diff
@@ -206,6 +214,7 @@ func (r *YurtClusterReconciler) runNodeConvert(ctx context.Context, nodeType ope
 	}
 
 	klog.Infof("run convert with result: \n%v", string(result))
+	r.recorder.Eventf(yurtCluster, corev1.EventTypeNormal, "ConvertCompleted", "edge node %s convert with result: %v.", r.Options.NodeName, string(result))
```

Comment:
The result message will be too long and will exceed the length limit of Kubernetes Event.